gensim/models/_fasttext_bin.py

```diff
@@ -197,6 +194,41 @@ def _load_vocab(fin, new_format, encoding='utf-8'):
     return raw_vocab, vocab_size, nwords
 
 
+def _decode_bad_unicode_py2(word_bytes, encoding):
+    #
+    # We are explicitly ignoring the encoding here.  This is bad, because the
+    # output is highly likely to be garbage, but its the only thing that:
+    #
+    # - does not cause collisions (see https://github.com/RaRe-Technologies/gensim/issues/2402)
+    # - works under Py2
+    #
+    del encoding
+    word = word_bytes.decode('latin1')
+    logger.error(
+        'failed to decode invalid unicode bytes %r; falling back to latin1, using %r; '
+        'consider upgrading to Python 3 for better Unicode handling (and so much more)',
+        word_bytes, word
+    )
+    return word
+
+
+def _decode_bad_unicode_py3(word_bytes, encoding):
```

Comment:
Docstring please.

Reply:
Reverted in favor of backslashreplace backport.